frontend/src/router.ts

```diff
@@ -93,7 +93,7 @@ export const router = new Router({
               component: () => import("@/views/TE/Upload.vue"),
               meta: {
                 menuItem: true,
-                auth: { requireClaims: { TE: true } },
+                auth: { requireClaims: { TE: true, coordinator: true } },
```

Comment:
Doesn't this require both? It should require any.

Reply:
The current way to check for claims requires any not both.